core/storage/skill/gae_models_test.py

```diff
@@ -57,43 +89,60 @@ def setUp(self):
 
         skill_models.SkillRightsModel(
             id='id_1',
-            creator_id='janet',
+            creator_id='janet_id',
             skill_is_private=True
         ).commit(
-            'janet', 'Created new skill rights',
+            'janet_commit_id', 'Created new skill rights',
             [{'cmd': rights_manager.CMD_CREATE_NEW}])
         skill_models.SkillRightsModel(
             id='id_2',
-            creator_id='janet',
+            creator_id='janet_id',
             skill_is_private=True
         ).commit(
-            'janet', 'Edited skill rights',
+            'janet_commit_id', 'Edited skill rights',
             [{'cmd': rights_manager.CMD_CHANGE_ROLE}])
         skill_models.SkillRightsModel(
             id='id_3',
-            creator_id='joe',
+            creator_id='joe_id',
             skill_is_private=False
         ).commit(
-            'joe', 'Created new skill rights',
+            'joe_commit_id', 'Created new skill rights',
             [{'cmd': rights_manager.CMD_CREATE_NEW}])
         skill_models.SkillRightsModel(
             id='id_4',
-            creator_id='joe',
+            creator_id='joe_id',
             skill_is_private=True
         ).commit(
-            'joe', 'Created new skill rights',
+            'joe_commit_id', 'Created new skill rights',
             [{'cmd': rights_manager.CMD_CREATE_NEW}])
 
+    def test_has_reference_to_user_id(self):
+        self.assertTrue(
+            skill_models.SkillRightsModel
+            .has_reference_to_user_id('janet_id'))
+        self.assertTrue(
+            skill_models.SkillRightsModel
+            .has_reference_to_user_id('janet_commit_id'))
```

Comment:
This (and some of the other surrounding lines) seems strange to me. Shouldn't the argument to has_reference_to_user_id be a user ID (like, I assume, janet_id)? "janet_commit_id" sounds like the ID of a commit.

Reply:
Yes, these should be `janet_committer_id` and `joe_committer_id`.

Reply:
I still don't understand why 'janet_committer_id' differs from 'janet_id'. If the user ID represents "Janet", the same ID should be used in the different places, right?

Reply:
It is different just to test if we can detect both `commmitter_id` and `creator_id` separately.